unix/vncpasswd/vncpasswd.cxx

```diff
@@ -133,6 +187,11 @@ static std::vector<uint8_t> readpassword() {
 
 int main(int argc, char** argv)
 {
+  setlocale(LC_ALL, "");
+  bindtextdomain(PACKAGE_NAME, LOCALE_DIR);
+  bind_textdomain_codeset(PACKAGE_NAME2, "UTF-8");
```

Comment:
This seems wrong. Why is this needed? We should not need to mess with a library's translation handling. And UTF-8 might not be the correct character set for the terminal.